statistics-production/user-eng.qmd

```diff
@@ -10,29 +10,31 @@
+The [Code of Practice for Statistics](https://code.statisticsauthority.gov.uk/) states that:
 
-The [Code of Practice for Statistics](https://code.statisticsauthority.gov.uk/) states that “users of statistics and data should be at the centre of statistical production; their needs should be understood, their views sought and acted on, and their use of statistics supported”. The quality pillar of the Code of Practice also reminds us that statistics should fit their intended uses. Producers of statistics will only know whether this element of quality continues to be achieved by regularly engaging with a full range of users.
+*“users of statistics and data should be at the centre of statistical production; their needs should be understood, their views sought and acted on, and their use of statistics supported”*. 
+
+Producers of statistics will only know whether this element of quality continues to be achieved by regularly engaging with a full range of users.
 
 ---
 
 ## USER hub
 
 
-The USER hub is now available on the [GSS website](https://gss.civilservice.gov.uk/user-support-and-engagement-resource-hub/), this is a central resource to help producers of statistics develop the knowledge, skills and techniques needed to engage effectively with their audiences.
+The User Support and Engagement Resource (USER) hub is now available on the [GSS website](https://gss.civilservice.gov.uk/user-support-and-engagement-resource-hub/). This is a central resource to help producers of statistics develop the knowledge, skills and techniques needed to engage effectively with their audiences.
 
 ---
 
 ## Improving your own engagmenent
 
 
-Download this [top tips document](../resources/2021_05_14_Top_Tips.docx){target="_blank" rel="noopener noreferrer"} produced for the GSS, as a starting point for understanding how you can improve your user engagement.
+Download this [top tips document](../resources/2021_05_14_Top_Tips.docx){target="_blank" rel="noopener noreferrer"} as a starting point for understanding how you can improve your user engagement.
 
-Through our work engaging users on EES we have built up a list of interested users who have stated a willingness to be contacted about opportunities to be involved in user testing of our statistics on the platform, if you have any suggestions for things for us to test on EES with these users, please contact [explore.statistics@education.gov.uk](mailto:explore.statistics@education.gov.uk).
+If you have any suggestions of things for us to test on EES with users who have volunteered to do user testing, please contact [explore.statistics@education.gov.uk](mailto:explore.statistics@education.gov.uk).
 
 More advice and guidance will follow in this space. In the meantime, if you're interested in user engagement please contact Heather Brown.
```

Comment:
should we just merge this last line with the one above and point them to the EES mailbox as Heather has left DfE now?

Reply:
done!